performance-testing/locust_run.py

```diff
@@ -27,7 +28,7 @@
 SCHEDULES = ["A"]  # Further schedules to be implemented in the future
 
 # Avoid "Too many open files" error
-resource.setrlimit(resource.RLIMIT_NOFILE, (10000, 999999))
+# resource.setrlimit(resource.RLIMIT_NOFILE, (10000, 999999))
```

Comment:
maybe we can remove this if no longer needed

Reply:
This line wouldn't work in my environment, but it also wasn't needed for locust to work.  I didn't want to remove it without knowing what it was intended to do, though

Reply:
@Elaine-Krauss-TCG thanks for taking a look. This is legacy from openFEC - let's leave it commented out as a breadcrumb in case someone else has a different issue.

Reply:
After further discussion, we agreed it's fine to take it out for now.